src/PhpSpec/Extension.php

```diff
@@ -13,14 +13,16 @@
 
 namespace PhpSpec;
 
+use UltraLite\Container\Container;
```

Comment:
That's true, although the PR isn't quite at the point where we can remove this interim measure just yet.  This is because I haven't put a composite container.  Hoping to get there very soon though.